must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -243,6 +243,7 @@ for ns in $namespaces; do
             { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd trash ls --pool $bp --format=json" >> "${COMMAND_OUTPUT_FILE}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-trash-ls-"${bp}"-json-debug.log 2>&1 &
         done
 
+        pids_rbd=()
```

Comment:
This should probably be done just before the `for image in $images; do` start of the loop. Otherwise, once the inner loop finishes, a wait for PIDs from the previous blockpool is repeated.